### ISSUE TITLE
Quick Help 

### ISSUE THREAD

**Issue by jonthegreat** [RocketMod/Rocket#589]
ok i need help from some1 who is a little more familiar with coding unturned servers i made a rank yet i cant make anyone that rank here is the code:
https://gyazo.com/d543f02fbd46c66aa352d14d87cd3065

thanks!

**Comment by Larsdude** [comment]
The <ParentGroup> should be either "default" or "VIP". That is the group which your new one is a subsidiary of. So if you put "default" in <ParentGroup>, then the "default" group would be the parent of your new "dev" group. The "dev" group then inherits all the <Commands> of it's parent group, "default", plus whatever commands are added in by the group's <Commands> section. The only thing I'm unsure of is whether the color permission will be overridden. The default color of text is white, so to be safe you could remove this line from the "default" group and nothing would change about it:
<Command>color.white</Command>

Hope this helps!

tl;dr
Change <ParentGroup>dev</ParentGroup>
to <ParentGroup>default</ParentGroup>
